### PR TITLE
Solved the purchase link issue

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
1) Added the new link creating mechanism.
2) Added the test.

### CODE REVIEW

**Review by kiroj305 — APPROVED** [review]

File: lib/utils/file_utils.dart [hunk 1]
```diff
@@ -51,4 +51,17 @@ class FileUtils {
...
         suffixes[i];
   }
+
+  static String generateEaselLink({required String recipeId, required String cookbookId}) {
+    return Uri.https('pylons.page.link', "/", {
+      "amv": "1",
+      "apn": "tech.pylons.wallet",
+      "ibi": "xyz.pylons.wallet",
+      "imv": "1",
+      "link": "https://wallet.pylons.tech/?action=purchase_nft&recipe_id=$recipeId&cookbook_id=$cookbookId&nft_amount=1"
```
Comment (mustafapylons) [inline lib/utils/file_utils.dart]: We should sanitize URLs like this by using the package below for XSS scripting:
https://pub.dev/packages/sanitize_html

**Review by mustafapylons — COMMENTED** [review]
We should use something like this to make sure there's no XSS/cross site scripting:
https://pub.dev/packages/sanitize_html

**Comment by kjawadDeveloper2** [comment]
@mustafapylons  can you check it please.

**Review by mustafapylons — APPROVED** [review]
Better, more secure code using sanitize package, thanks.

**Pull request merged**